### ISSUE TITLE
Add streaming video from mobile phone camera

### ISSUE THREAD

**Issue by grishakonzalaev** [BearWare/TeamTalk5#440]
it would be cool to stream video from mobile phone camera.

**Comment by slannon97** [comment]
I completely agree! The mobile versions of the client should be able to send and receive video data. Why they currently cannot is beyond me. This is useful if you are talking with sighted people who will most likely want two-way video.

**Comment by user2200** [comment]
Полностью поддерживаю Потоковое видео очень нужно.

**Comment by jsl303** [comment]
Any update? It would be very nice for mobile devices to transmit videos.
